Dynamo/Dynamo.SwiftLang/SLAttribute.cs

```diff
@@ -11,12 +12,21 @@ public SLAttribute (SLIdentifier name, CommaListElementCollection<SLBaseExpr> ar
 		{
 			Name = Exceptions.ThrowOnNull (name, nameof (name));
 			Add (new SimpleElememt ("@"));
+			var stringRep = new StringBuilder (Name.Name);
 			Add (Name);
 			if (args != null) {
 				Add (new SimpleElememt ("("));
 				Add (args);
 				Add (new SimpleElememt (")"));
+				stringRep.Append ("(");
+				foreach (var arg in args) {
```

Comment:
I was going to suggest being cute here with a one liner in linq, but StringBuilder is lame and doesn't take a IEnumerable AppendRange so this is fine.